frontend/components/feedback-task/SearchBar.base.vue

```diff
@@ -0,0 +1,138 @@
+<!--
+  - coding=utf-8
+  - Copyright 2021-present, the Recognai S.L. team.
+  -
+  - Licensed under the Apache License, Version 2.0 (the "License");
+  - you may not use this file except in compliance with the License.
+  - You may obtain a copy of the License at
+  -
+  -     http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing, software
+  - distributed under the License is distributed on an "AS IS" BASIS,
+  - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  - See the License for the specific language governing permissions and
+  - limitations under the License.
+  -->
+
+<template>
+  <form @submit.prevent="searchText(query)">
+    <div class="searchbar" :class="{ active: query }" :style="searchBarStyles">
+      <BaseIcon
+        v-if="!query"
+        icon-name="search"
+        icon-width="20"
+        icon-height="20"
+      />
+      <BaseButton
+        v-else
+        class="searchbar__button"
+        @click="removeCurrentSearchText()"
+      >
+        <BaseIcon
+          class="searchbar__button__icon"
+          icon-name="close"
+          icon-width="20"
+          icon-height="20"
+        />
+      </BaseButton>
+      <label class="searchbar__label" for="query" v-text="description" />
+      <input
+        ref="input"
+        class="searchbar__input"
+        type="text"
+        name="query"
+        id="query"
+        v-model="query"
```

Comment:
Add lazy mode to v-model to not fire reactivity on every change : 
before 
```suggestion
        v-model.lazy="query"
```